docs/guides/admin/docs/workflowoperationhandlers/cropvideo-woh.md

```diff
@@ -0,0 +1,26 @@
+# Crop Workflow operation
+
+The plugin provides the workflow operation `crop-video`. This workflow operation excutes ffmpeg command `cropdetect`.
+`cropdetect` checks for black bars on the sides of the track of the workflow instance. If `cropdetect` is successfully,
```

Comment:
successfully -> successful

Reply:
Surely Michael ;-)
Thanks for the review.